src/components/ProductForm/ProductForm.jsx

```diff
@@ -26,12 +27,23 @@ export default function ProductForm({
     initialValues?.price ? getPrice(initialValues.price) : '',
   );
   const [error, setError] = useState('');
+  const [designers, setDesigners] = useState([]);
+  const [selectedDesigner, setSelectedDesigner] = useState('');
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    // maybe I can ditched this GET request and utilize designer and setDesigner (initialValues)
+    api.get('/designer').then(({ data }) => {
+      setDesigners(data);
+      setLoading(false); // not sure if needed, copied from users example in OrderCart.jsx
+    });
+  }, []);
 
   const validate = (data) => {
     const schema = yup.object().shape({
       name: yup.string().required(),
       description: yup.string(),
-      designer: yup.string(),
+      designer_id: yup.number(), // need to be updated to designer_id?
```

Comment:
Yep! Whatever we want on a form

Reply:
ah...ya, I made this comment before I decided to change. I'll remove comment before next commit.